libs/platform/experience/src/services/layout/default-layout.builder.ts

```diff
@@ -135,41 +125,67 @@ export class DefaultLayoutBuilder implements LayoutBuilder {
 
     add(
       {
-        '--cols': data.columnCount,
-        '--grid-column': data.gridColumn,
-        '--grid-row': data.gridRow,
-        '--span': data.span,
-        '--z-index': data.zIndex,
-        '--rotate': data.rotate,
+        // '--cols': data.columnCount,
```

Comment:
Do we need it?

Reply:
no, we replaced it with --col-pos and --row-pos